workspace-manager-clienttests/src/main/java/scripts/utils/WorkspaceTestScriptBase.java

```diff
@@ -0,0 +1,67 @@
+package scripts.utils;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.client.ApiException;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class WorkspaceTestScriptBase extends TestScript {
```

Comment:
Can you add javadocs, especially if this is going to be the base for a number of tests?